extensions/amp-sticky-ad/0.1/amp-sticky-ad.css

```diff
@@ -40,3 +40,13 @@ amp-sticky-ad {
 .-amp-sticky-ad-layout > amp-ad {
   display: block;
 }
+
+.-amp-sticky-ad-close-button {
+  position: absolute;
+  height: 33px;
+  width: 33px;
+  background-color: black;
+  opacity: 0.5;
+  right: 0px;
```

Comment:
Let's ensure that it's aligned to right by default and to the left for `dir=rtl`